vocabularies/EntityRelationship.xml

```diff
@@ -0,0 +1,292 @@
+            <PropertyValue Property="href" String="https://github.com/sap/odata-vocabularies/blob/main/vocabularies/EntityRelationship.md" />
+          </Record>
+        </Collection>
+      </Annotation>
+
+      <Term Name="entityType" Type="Edm.String" Nullable="false" AppliesTo="EntityType">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" >
+          <String>
+            Defines which Entity Type the current JSON Schema object represents. 
+            There could be several JSON Schema objects that are assigned to the same Entity Type. 
+            One JSON Schema object can only have one Entity Type assigned, which corresponds to the 
+            applications own Entity Type definition. The ID scheme for an Entity Type ID is as following: 
+            entityTypeId := namespace:entityTypeLocalId[:v"majorVersion"]
+          </String>
+        </Annotation>
+      </Term>
+
+      <Term Name="propertyType" Type="Edm.String" Nullable="false" AppliesTo="Property">
+        <Annotation Term="Common.Experimental" />      
+        <Annotation Term="Core.Description" String="logical property type of a property" />
+        <Annotation Term="Core.LongDescription">
+          <String>Defines the logical Property Type of a property.
+            The reason is to have an ID to relate to the property type independently of the property name and shared 
+            across every usage of the same property across APIs where names might be different. 
+            Especially to state that it can be used as an ID or is part of a composite ID.
+        </String>
+        </Annotation>
+      </Term>
+
+      <Term Name="entityIds" Type="Collection(EntityRelationship.entityId)" Nullable="false" AppliesTo="EntityType">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="The annotated entity type represents an ODM root entity" />
+        <Annotation Term="Core.LongDescription">
+          <String>
+            Defines a list of IDs, which are available to look up the Entity Type or create a reference to it.
+            An Entity Type can have multiple IDs:
+            * There can be **alternative IDs** that can also be used to create a reference to the Entity Type
+            * Some ID properties form a **composite ID** together and need to be combined to function as a unique ID for references
+            * This is indicated by a propertyTypes array with more than one Property Type ID entry.          
+          </String>
+        </Annotation>
+      </Term>
+
+      <Term Name="reference" Type="EntityRelationship.singleReference" Nullable="false" AppliesTo="Property">
+        <Annotation Term="Core.Description" String="      Defines references to other Entity Types based on a single ID." />
+        <Annotation Term="Core.LongDescription">
+          <String>      Defines references to other Entity Types based on a single ID.
+          </String>
```

Comment:
Please don't repeat the Description verbatim in the LongDescription. If you don't have a longer description, just omit it.

Also applies to the next five terms. Aren't there longer descriptions available for these?

Reply:
+1